proposals/NNNN-NextRpcIndication.md

```diff
@@ -0,0 +1,98 @@
+# Next RPC Indication for the HMI
+
+* Proposal: [SDL-NNNN](NNNN-NextRPCIndication.md)
+* SDL Proposal: [To be filled later](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/)
+* Author: [Michael Crimando](https://github.com/MichaelCrimando)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / Web / RPC ]
+
+## Introduction
+
+In situations with driver distraction, also with making a good UI, it really hurts the possible UX when the HMI has no idea what's going to happen next after the user taps an in-app menu item, a softbutton or a selection in a perform interaction. 
+This feature would provide Apps a way for the HMI to know what's next when a button or choice is selected.
+
+## Motivation
+
+As a system, I want to be able to be smarter and faster with in-app menu systems.
+This enables:
+1. Better HMI possible. Can show forward and backwards arrows for the user so the user knows what to expect when navigating around.
+2. Smart Driver Distraction. If a softbutton or choice would lead to a screen that's normally locked out while driving (like a Scrollable Message), the HMI could show the softbutton as disable while driving.
```

Comment:
```suggestion
2. **Smart Driver Distraction:** if a softbutton or choice would lead to a screen that's normally locked out while driving (like a Scrollable Message), the HMI could show the softbutton as disabled while driving.
```